applications/index.html

```diff
@@ -78,7 +78,7 @@ <h2>Status</h2>
         <div class="application-details-main"></div>
         <div class="application-details-actions">
           <button class="application-details-accept">Accept</button>
-          <p class="application-details-reject">Reject</p>
+          <button class="application-details-reject">Reject</button>
```

Comment:
Also, you can consider keyboard focus states (`:focus-visible`) for both buttons so users relying on keyboards can easily navigate.

Reply:
Thanks for pointing out! updated.